### PR TITLE
Some additional entries for Supervised Learning and Statistics sections

### PR DESCRIPTION
**Pull request by alexmalins**
- Some additional entries for Supervised Learning and Statistics sections
- Also sort the Algebra and Calculus section alphabetically

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -204,25 +204,30 @@ This is an English-Japanese lexicon for Machine Learning and Deep Learning termi
...
 | Classification | 分類 |
 | Classification and Regression Trees (CART) | 分類・回帰ツリー (CART) |
 | Classifier | 分類器 |
 | Closed form solution | 閉形式の解 |
 | Coefficients | 係数 |
+| Confusion matrix | 混同行列 |
 | Continuous values | 連続値 |
 | Cost function | コスト関数 |
 | Cross-entropy | クロスエントロピー |
+| Cross validation | クロスバリデーション |
```
Comment (ytknzw) [inline README.md]:
```suggestion
| Cross validation | 交差検証 / クロスバリデーション|
```

File: README.md [hunk 2]
```diff
@@ -393,38 +407,38 @@ This is an English-Japanese lexicon for Machine Learning and Deep Learning termi
...
 | Diagonal | 対角成 |
-| Trace | 跡 |
+| Element | 要素 |
+| Function | 関数 |
+| Invertible | 可逆 |
+| Linear Algebra          | 線形代数                   |
+| Matrix          | 行列                    |
+| Norm | ノルム |
+| Notation          | 表記法                   |
+| Row          | 行目                |
```
Comment (ytknzw) [inline README.md]:
```suggestion
| Row          | 行                |
```

File: README.md [hunk 3]
```diff
@@ -393,38 +407,38 @@ This is an English-Japanese lexicon for Machine Learning and Deep Learning termi
...
 
 | English            | 日本語                 |
 |:-------------------|:-----------------------|
-| Linear Algebra          | 線形代数                   |
+| Antisymmetric | 反対称 |
 | Calculus          | 微積分                   |
-| Vector          | ベクトル                   |
-| Matrix          | 行列                    |
-| Row          | 行目                |
 | Column          | 列目                   |
```
Comment (ytknzw) [inline README.md]:
```suggestion
| Column          | 列                   |
```

**Review by ytknzw — CHANGES_REQUESTED** [review]

**Comment by alexmalins** [comment]
Thanks!

**Comment by suzana-ilic** [comment]
Thank you both for your contributions!

**Pull request merged**